<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnotateWindow</class>
 <widget class="QMainWindow" name="AnnotateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnnotateWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GraphicsAnnotator" name="Annotate2D"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="LabelPosX">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LabelPosY">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LabelFilename">
           <property name="text">
            <string>Filename</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="BtnSetImageDir">
         <property name="text">
          <string>载入图片</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="BtnFormerImage">
           <property name="text">
            <string>上一张</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnNextImage">
           <property name="text">
            <string>下一张</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnFinish">
           <property name="text">
            <string>标注完成(当前帧)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="BtnSave">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsAnnotator</class>
   <extends>QGraphicsView</extends>
   <header>GraphicsAnnotator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
